dts/bindings/sensor/adi,adxl345-common.yaml

```diff
@@ -35,7 +35,7 @@ properties:
     type: int
     description: |
       Specify the FIFO watermark level in frame count.
-      Valid range: 1 - 31
+      Valid range: 2 - 31
```

Comment:
Is setting `fifo-watemark = <1>;` an invalid or unsupported configuration by the sensor? I don't understand the value of limiting the driver, beyond its capabilities; irrespective of having a clear-cut use-case.